contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -331,4 +331,33 @@ contract MultipleArbitrableTransaction {
     function getCountTransactions() public view returns (uint countTransactions) {
         return transactions.length;
     }
+
+    /** @dev Getter to get transaction ids by an address.
+     *  for the arbitrable transaction with the id 0, the GUI must checks
+     *  if the transaction is a "false positive" or not
+     *  @param _address of the owner of the transactions.
+     *  @return array of transaction ids.
+     */
+    function getTransactionIdsByAddress(address _address) public view returns (uint[]) {
+        uint[] memory arbitrableTransactionIds = new uint[](transactions.length);
```

Comment:
I prefer don't use the underscore `_` for a memory variable (`_count`, `_transactionIDs`) because in general it's reserved for the parameters like `_address` (convention).